### ISSUE TITLE
Set up Mirror Sites of services (Trac #1239)

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#1265]
Migrated from http://trac.sasview.org/ticket/1239
```json
{
    "status": "new",
    "changetime": "2019-02-27T17:43:06",
    "_ts": "2019-02-27 17:43:06.806007+00:00",
    "description": "AS part of our effort to increase robustness, it would be nice to have mirror sites with automatic rollover for our \"privately managed\" cloud resources so that we are not blocked by the currently rather rare instances of service failures (power outages at UTK or DMSC, disk crashes, etc).  In a first instance mirroring all the services at UTK at 2 other sites should be an achievable goal.  If purchased right and if the build servers and Jenkins system at DMSC can be put into VMs/containers it might be possible to get the DMSC services mirrored using the same hardware?\n\nSteps needed:\n * Design system (how will it work)\n * Identify hardware needed\n * Identify location to house hardware\n * Identify funding for hardware\n * purchase and install hardware\n * set up mirroring",
    "reporter": "butler",
    "cc": "",
    "resolution": "",
    "workpackage": "Support Infrastructure",
    "time": "2019-02-27T17:43:06",
    "component": "SasView",
    "summary": "Set up Mirror Sites of services",
    "priority": "major",
    "keywords": "",
    "milestone": "Admin Tasks",
    "owner": "",
    "type": "defect"
}
```
